book/go/api/api.md

```diff
@@ -267,13 +267,13 @@ A common encoding used in Wallaroo is a big-endian 32-bit unsinged integer, whic
 
 ##### `Decode(b []byte) interface{}`
 
-Return a Go object of the type the next step in the pipeline expects.
+Return a Go object of the type the next step in the pipeline expects or `nil` in order to filter out the message before sending to the next step.
```

Comment:
Instead of 
> or `nil` in order to filter out the message before sending to the next step
I would say
> or `nil` in order to filter out the message instead of sending to the next step